### ISSUE TITLE
[Card - pictogram] Design updates to improve mobile accessibility 

### ISSUE THREAD

**Issue by shixiedesign** [carbon-design-system/carbon-for-ibm-dotcom#6739]
### Problem

[**Card - pictogram** component ](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-card--pictogram) has a few issues: 
- When pictogram is bottom, a hover interaction reveals additional body copy. On mobile (no hover state), these additional body copy is never readable. We should have a new mobile design so the body copy is always revealed
- When the body copy and card headline are long, hover interaction makes the card grow taller. This is probably because the card size is not calculated with those elements present at first. Might need a refactor? 

### Task 
1. Look into additional of the designs for mobile design 
2. Create logic for how the content edge cases should be handled

### Additional information
- Design spec: https://ibm.ent.box.com/folder/114042319288
- Functional spec: https://ibm.box.com/s/hgvu48r1fv2ga2h0yn3qshrtxjc71vch

**Comment by RobertaJHahn** [comment]
React version is behind a feature flag.  React component will not be updated.  No issues needed.
https://ibmdotcom-react-experimental.mybluemix.net/iframe.html?id=components-card--default&knob-Show%20pictogram%20(hides%20CTA)_Card=true&knob-Card%20Eyebrow_Card=Eyebrow%20text&knob-Card%20Heading_Card=Lorem%20ipsum%20dolor%20sit%20amet&knob-Card%20body%20copy_Card=&knob-CTA%20copy_Card=Card%20CTA%20text

**Comment by RobertaJHahn** [comment]
Jeff will include agenda item in the engineering Huddle to talk about deprecating the Pure React version of Card - pictogram component.

**Comment by shixiedesign** [comment]
Re: deprecating Pure react card pictogram @jeffchew @RobertaJHahn 
1. I believe IBM Docs is using currently, so should we clear it with them first? 
2. if we can get the Web component card pictogram fixed first, that would make it easier for deprecation first I hope

**Comment by skuruvil** [comment]
Design spec: https://ibm.ent.box.com/folder/114042319288
Functional spec: https://ibm.box.com/s/hgvu48r1fv2ga2h0yn3qshrtxjc71vch

**Comment by oliviaflory** [comment]
This epic is complete
